src/components/Card.js

```diff
@@ -1,9 +1,23 @@
 export default class Card {
-  constructor( data, cardSelector, handleImageClick) {
+  constructor(
+    data,
```

Comment:
Needs correction:  I can't remove the new added cards with the error `Invalid cardId format. Must be a valid ObjectId`.